civihr_default_theme/assets/js/civihr_default_theme.script.js

```diff
@@ -133,6 +144,23 @@
   }
 
   /**
+   * Add Images in Customize Onboarding Page
+   */
+  function addImagesInCustomizeOnboardingPage () {
+    if (!$('.onboarding-customize-logo').length) {
+      $('#edit-civihr-onboarding-organization-logo-fid-ajax-wrapper').before('<img class="onboarding-customize-logo" src="../../' + Drupal.settings.civihr_default_theme.path +'/assets/images/onboarding_wizard/logo-img.jpg"/>');
```

Comment:
Seems like `./../' + Drupal.settings.civihr_default_theme.path +'/assets/images/` could be extracted to a variable like `imagesDirectory` or something